decentraland/helpers.js

```diff
@@ -163,6 +164,17 @@ export const checkBlockchain = async () => {
   return true;
 };
 
+export const formatNumber = number => {
+  if (Platform.OS === "android") {
+    // only android needs polyfill
+    require("intl");
+    require("intl/locale-data/jsonp/en-US");
+  }
+  const formatter = new Intl.NumberFormat("en-US");
```

Comment:
```suggestion
  // TODO: Handle internationalization for other regions
  const formatter = new Intl.NumberFormat("en-US");
```

Reply:
Done.

Reply:
Thanks! Out of curiosity, why not accept the suggestion through the GitHub UI?

Reply:
UI suggestion commit doesn't work if the file has changed after the suggestions, I've changed this file.

Reply:
Ah ok, good to know!

Reply:
Resolving this